Source/Core/VideoBackends/D3D12/D3D12BoundingBox.cpp

```diff
@@ -2,10 +2,10 @@
 // Licensed under GPLv2+
 // Refer to the license.txt file included.
 
-#include "VideoBackends/D3D12/BoundingBox.h"
+#include "VideoBackends/D3D12/D3D12BoundingBox.h"
```

Comment:
This should go just before D3D12Renderer

Reply:
The header file of the current cpp is always first regardless of name.  That's how the linter works

Reply:
Got it